@types/jsb.d.ts

```diff
@@ -339,6 +339,10 @@ declare namespace jsb {
         setVerifyCallback (verifyCallback: (path: string, asset: ManifestAsset) => boolean): void;
         setEventCallback (eventCallback: (event: EventAssetsManager) => void): void;
     }
+
+    export class JSBNativeDataHolder {
+
+    }
```

Comment:
Why export it? Does developers need to use it?

Reply:
In order to declare the type, if the type is not declared, the return would need to be of type any.

Reply:
But we don't want to export it to developers. So should find a way to export internally.

Reply:
The jsb namespace is deprecated and will be used internally as an engine.